features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/statistics/usage.jag

```diff
@@ -1,14 +1,21 @@
 <%
 log = new Log(),
 apiStore = jagg.module("manager").getAPIStoreObj(),
+usageClient = jagg.module("manager").getUsageClient(),
 user = jagg.getUser();
-
+var gson=com.google.gson.Gson();
 
 function getFirstAccessTime(providerName, server) {
     var usage = [];
     var usageout = [];
     try {
-        usage = apiStore.getFirstAccessTime(providerName, server);
+        if(usageClient==null){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea